tf2onnx/rewriter/lstm_rewriter.py

```diff
@@ -73,7 +73,7 @@ def get_weight_and_bias(self, match):
         if not ft:
             return None
 
-        if not (ft.value == 1 and self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0])):
+        if not self.g.get_dtype(b_e.output[0]) == self.g.get_dtype(ft_bias.output[0]):
```

Comment:
maybe we should check ft_bias being a scalar?

Reply:
Sure! using `np.ndim(ft.value) == 0` to check whether ft is a scalar